rosa_hcp/rosa-hcp-cluster-no-cni.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="rosa-hcp-cluster-no-cli"]
+= {hcp-title} clusters without a CNI plugin
+include::_attributes/attributes-openshift-dedicated.adoc[]
+include::_attributes/common-attributes.adoc[]
+:context: rosa-hcp-cluster-no-cni
+
+toc::[]
+
+Customers can use their own Container Network Interface (CNI) plugin when creating a {hcp-title-first} cluster.
+You can create a {hcp-title} cluster without a CNI in order to bring your own CNI plugin and install it after cluster creation.
+
+[NOTE]
+====
+For clusters in this configuration, the responsibility of CNI plugin support belongs to the customer in coordination with their chosen CNI vendor.
+====
+
+[id="rosa-hcp-no-cni-cluster-creation"]
+== Creating a {hcp-title} cluster without a CNI plugin
+
+=== Prerequisites
+* Ensure that you have completed the xref:../rosa_planning/rosa-sts-aws-prereqs.adoc[AWS prerequisites].
+
+To create a {hcp-title} cluster, you must have the following items:
+
+* A configured virtual private cloud (VPC)
+* Account-wide roles
+* An OIDC configuration
+* Operator roles
+
+[id="rosa-hcp-creating-vpc-no-cni"]
+=== Creating a Virtual Private Cloud for your {hcp-title} clusters
+
+You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. You can use the following methods to create a VPC:
```

Comment:
There are many more methods to create a VPC. The 2 listed aren't comprehensive. So I suggest writing this accordingly. And, leave out the section for creating VPC using TF because a) this is not core to this topic or Rosa with HCP or OpenShift in general. I'd suggest leaving this topic in the Tutorials section and reference in a statement like this one:

You must have a VPC to create ROSA with HCP cluster. If you do not have one, you can following steps outlined in 'Creating a Virtual Private Cloud' [section](https://docs.openshift.com/rosa/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html#rosa-hcp-creating-vpc).

Reply:
The product assumption here and target customer/audience for ROSA with HCP or ROSA using BYO-VPC is that they have VPC that they want to use. If we are guiding users to create VPC, then they must just use the VPC ROSA Service creates. 
It's like a) people can prepare their meal. b) people can prepare the meal we sell. We are not teaching people to prepare their meal. Our business is a) selling software (OCP) and b) selling service (Managed OpenShift). We are not cloud consulting firm.